examples/gno.land/r/gnoland/faucet/admin.gno

```diff
@@ -5,44 +5,83 @@ import (
 	"std"
 )
 
-func AdminSetInPause(inPause bool) error {
+func AdminSetInPause(inPause bool) string {
 	if err := assertIsAdmin(); err != nil {
-		return err
+		return err.Error()
 	}
 	gInPause = inPause
-	return nil
+	return ""
 }
 
-func AdminSetMessage(message string) error {
+func AdminSetMessage(message string) string {
 	if err := assertIsAdmin(); err != nil {
-		return err
+		return err.Error()
 	}
 	gMessage = message
-	return nil
+	return ""
 }
 
-func AdminSetPerTransferSend(send std.Coins) error {
+func AdminSetTransferLimit(amount int64, denom string) string {
 	if err := assertIsAdmin(); err != nil {
-		return err
+		return err.Error()
 	}
-	gPerTransferSend = send
-	return nil
+	gLimit = std.Coins{std.Coin{Denom: denom, Amount: amount}}
```

Comment:
Since this PR breaks the multi-coins feature, let's replace the global var from `std.Coins` with `std.Coin` and completely hide the denom argument.

Reply:
The Bank module and std.Bank's SendCoins and GetCoins API take Coins only.  Can you celebrate more about how this breaks the multi-coins feature?

Reply:
The previous call was accepting std.Coins as parameters made it possible to pass not only one coin but a collection of coins: `50ugnot + 50usomething`. It's not possible anymore.

Your PR hardcode `ugnot` at various places in the code, so let's remove the last one here because if I want to use a custom denom here, the rest of the code won't follow.

I'm okay to switch to an ugnot-only faucet, but then we can make it more explicit about storing simple variables like `gLImit = std.Coin{Denom:"ugnot", Amount: amount}` and pass `std.Coins{gLimit}` for `SendCoins` and `GetCoins` calls.

Reply:
> The previous call was accepting std.Coins as parameters made it possible to pass not only one coin but a collection of coins: `50ugnot + 50usomething`. It's not possible anymore.
> 
> Your PR hardcode `ugnot` at various places in the code, so let's remove the last one here because if I want to use a custom denom here, the rest of the code won't follow.
> 
> I'm okay to switch to an ugnot-only faucet, but then we can make it more explicit about storing 

Thank you for the explanation. Now I understand and will make changes.